README.rst

```diff
@@ -0,0 +1,77 @@
+========
+Workflow
+========
+
+This is the MIT Libraries' `Airflow <https://airflow.apache.org/>`_ implementation. **Please read these instructions carefully, because this is very much a shared codebase.**
+
+.. contents:: Table of Contents
+.. section-numbering::
+
+Intro
+-----
+
+This repo contains a few different things. The main attraction is the ``workflows`` directory which contains all our Airflow DAGs. There is also the Docker infrastructure used for deploying, both locally for development and in production. Finally, there's a custom Python command line tool for performing various maintenace tasks on the Fargate cluster.
+
+Workflows
+---------
+
+The main thing to remember is that all our workflows go in the ``workflows`` directory and are deployed together. If you follow our usual GitHub practices, this shouldn't be a problem. A general overview of the process for publishing a new workflow will look like this:
+
+1. Create your new workflow and test it locally.
+2. Commit your changes and create a PR.
+3. Wait for the changes to be deployed in the staging cluster to verify they are working as expected.
+4. Get your PR approved and merged.
+
+Dependencies
+^^^^^^^^^^^^
+
+One thing that may be confusing at first is how dependencies are handled for a workflow. The answer is pretty simple, if somewhat unsastisfying: all workflows share the same dependencies. Everything goes in the same bucket. This is a function of how Airflow is built.
+
+When you need a new dependency for your workflow use ``pipenv`` the way you normally would::
+
+  $ pipenv install <dependency>
+
+It will be incumbent upon all of us to keep an eye on which dependency versions have changed in a PR and also ensure *that we have a good test suite for our workflows*.
```

Comment:
We'll probably want a checklist in our PR template that includes this type of info to help us remember. Mario is one repo I think we have this in.